src/browser/components/NewTeamModal.jsx

```diff
@@ -9,10 +9,6 @@ import {Modal, Button, FormGroup, FormControl, ControlLabel, HelpBlock} from 're
 import Utils from '../../utils/util';
 
 export default class NewTeamModal extends React.Component {
-  static defaultProps = {
-    restoreFocus: true,
-  };
-
```

Comment:
`restoreFocus` is used by react-bootstrap's Modal component, or rather child of, to restore focus to the last focused element, which in our case may be a `webview`. Their library doesn't play well with focusing Electron webContents and this logic should be left to us.
The error that it generates has been hidden until now because after creating a new team the app was calling `loadURL` to completely reload `index.html`.